lib/zipkin-tracer/faraday/zipkin-tracer.rb

```diff
@@ -25,7 +25,7 @@ def call(env)
 
     private
 
-    SERVER_ADDRESS_SPECIAL_VALUE = '1'.freeze
+    SERVER_ADDRESS_SPECIAL_VALUE = true
```

Comment:
I assume thrift will encode this properly, but curious.. did you check?

Reply:
I think somewhere else we do a .to_s on the input, which probably will result in this not working as expected. might be wrong. This is why in #138 I dodged in a different way

Reply:
Thanks! I'll fix not to do `.to_s` on booleans.